updater_test.go

```diff
@@ -269,13 +269,11 @@ func TestUpdaterDownloadError(t *testing.T) {
 
 	require.NotNil(t, ctx.errReported)
 	assert.Equal(t, ctx.errReported.(Error).errorType, DownloadError)
-	assert.Equal(t, "deadbeef", ctx.updateReported.InstallID)
```

Comment:
You're only going to have `updateReported` if the user's chosen an action at the prompt, and here we get a download error, which happens before the prompt, so just remove this part of the test.